File: c/questao4.c
#include <stdio.h>

int main() {
    float kmPercorridos, diasAlugados, precoTotal;

    printf("Digite a quantidade de km percorridos: ");
    scanf("%f", &kmPercorridos);

    printf("Digite a quantidade de dias de aluguel: ");
    scanf("%f", &diasAlugados);

    precoTotal = (60 * diasAlugados) + (0.15 * kmPercorridos);

    printf("O preco a pagar pelo aluguel do carro é de R$%.2f\n", precoTotal);

    return 0;
}

File: python/questao4.py
km_percorridos = float(input("Digite a quantidade de km percorridos: "))

dias_alugados = int(input("Digite a quantidade de dias de aluguel: "))

preco_total = (60 * dias_alugados) + (0.15 * km_percorridos)

print(f"O preco a pagar pelo aluguel do carro é de R${preco_total:.2f}")
